collector/benchmarks/README.md

```diff
@@ -68,6 +68,8 @@ programs.
   caused [poor performance](https://github.com/rust-lang/rust/issues/32278) in
   the past.
 - **ctfe-stress-2**: A stress test for compile-time function evaluation.
+- **ctfe-uninit-stress**: A stress test for representation of values with
```

Comment:
In terms of naming I'd prefer `ctfe-stress-X`, as in `ctfe-stress-uninit`, for better grouping.